Samples/DecoderSample/SensorDecoderModule.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Generally speaking we don't need to follow the same SDK upgrades for the samples/CLI as with the Core modules. For example, the cayenne module hasn't been updated since .NET Core 2.1, hence I thought that I will share the target framework only where necessary.

Reply:
Addressed in d77c790ccd3d7793c7a3f8108531f5d205269b97